### PR TITLE
Release grid menus

### PR DESCRIPTION
**Pull request by orangejenny**
Verified with @dimagi/product 

@mkangia 

fyi @wpride

### CODE REVIEW

**Comment by wpride** [comment]
Nice! Its been a long road since the PID: https://github.com/dimagi/commcare-android/pull/89

**Comment by czue** [comment]
awesome!

File: corehq/apps/app_manager/static/app_manager/json/v1/commcare-app-settings.yaml [hunk 1]
```diff
@@ -146,19 +146,16 @@
 
 - id: use_grid_menus
   name: Modules Menu Display
-  description: Display root menu as a list or grid.
+  description: Display root menu as a list or grid. Read more on the <a target="_blank" href="https://confluence.dimagi.com/display/commcarepublic/Grid+View+for+Form+and+Module+Screens">Help Site</a>.
```
Comment (snopoke) [inline corehq/apps/app_manager/static/app_manager/json/v1/commcare-app-settings.yaml]: nit: should be `help.commcarehq.org`

Comment (snopoke) [inline corehq/apps/app_manager/static/app_manager/json/v1/commcare-app-settings.yaml]: also below

Comment (orangejenny) [inline corehq/apps/app_manager/static/app_manager/json/v1/commcare-app-settings.yaml]: thanks, fixed

**Pull request merged**